yt/frontends/arepo/data_structures.py

```diff
@@ -77,6 +77,10 @@ def _get_uvals(self):
 
     def _set_code_unit_attributes(self):
         self._unit_base = self._get_uvals()
+        if self._unit_base is not None and "UnitLength_in_cm" in self._unit_base:
+            # We assume this is comoving, because in the absence of comoving
+            # integration the redshift will be zero.
+            self._unit_base["cmcm"] = 1.0 / self._unit_base["UnitLength_in_cm"]
```

Comment:
Would it make sense to move this logic inside the `if unit in handle` block in the `_get_uvals` defintion? It seems like some of this logic is replicating that.

Reply:
I moved this inside `_get_uvals`--thanks for the suggestion!